src/test/scala/io/iohk/ethereum/mining/BlockGeneratorSpec.scala

```diff
@@ -111,6 +116,43 @@ class BlockGeneratorSpec extends FlatSpec with Matchers with PropertyChecks with
     fullBlock.right.foreach(b => b.body.transactionList shouldBe Seq(signedTransaction, nextTransaction))
   }
 
+  it should "filter out failing transaction from the middle of tx list" in new TestSetup {
+    val nextTransaction: SignedTransaction = SignedTransaction.sign(transaction.copy(nonce = signedTransaction.tx.nonce + 1), keyPair, Some(0x3d.toByte))
+
+    val privateKeyWithNoEthere = BigInt(1, Hex.decode("584a31be275195585603ddd05a53d16fae9deafba67213b6060cec9f16e44cae"))
+
+
+//    val gParam = new ECKeyGenerationParameters(curve, new SecureRandom())
+//    val eGen = new ECKeyPairGenerator
+//    eGen.init(gParam)
+//    val ephemPair = eGen.generateKeyPair
+//    val v = Hex.toHexString(ephemPair.getPrivate.asInstanceOf[ECPrivateKeyParameters].getD.toByteArray)
```

Comment:
Minor detail: Should we remove this commented code or maybe it's needed for something?